deis-tests-parallel/manifests/deis-tests-parallel-pod.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: deis-tests
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  restartPolicy: Never
+  containers:
+  - name: deis-e2e
+    image: quay.io/deisci/deis-e2e:canary
+    imagePullPolicy: Always
+    command: ["ginkgo"]
+    args: ["-slowSpecThreshold=120.00", "-nodes=30", "-noisyPendings=false", "-progress", "tests/"]
```

Comment:
did you try using "-stream" option..It's really helpful when running in parallel.
```
Setting stream to true will, instead, stream output from all parallel nodes in real-time, prepending each log line with the node # that emitted it
```